lib/kdump_utils.pm

```diff
@@ -103,6 +103,11 @@ sub activate_kdump {
                 record_soft_failure 'default kdump memory size is too small, kdumptool gets killed by OOM, bsc#1120566';
             }
         }
+        # enable and verify fadump settings
+        if (get_var('FADUMP') && check_screen('yast2-fadump-not-enabled')) {
```

Comment:
```suggestion
        if (get_var('FADUMP') ) {
```

Reply:
Not good idea, it would loop over `assert_screen 'yast2-fadump-enabled'` all the time.

Reply:
my concern here that the fact that `yast2-fadump-not-enabled` haven't match does not automatically means that we have `yast2-fadump-enabled` . So in current version of code we have a flow where variable FADUMP defined but fadump is actually not enabled . Is it ok for you ?

Reply:
Not sure if i understand, I just want to enable fadump by `alt-f` if it is requested by `FADUMP` and it is not enabled yet `yast2-fadump-not-enabled`.